util/src/main/java/org/killbill/billing/util/customfield/api/DefaultCustomFieldUserApi.java

```diff
@@ -38,21 +39,18 @@
 import org.killbill.billing.util.entity.dao.DefaultPaginationHelper.SourcePaginationBuilder;
 
 import com.google.common.base.Function;
-import com.google.common.collect.Collections2;
-import com.google.common.collect.ImmutableList;
-import com.google.common.collect.Iterables;
 import com.google.inject.Inject;
 
 import static org.killbill.billing.util.entity.dao.DefaultPaginationHelper.getEntityPaginationNoException;
 
 public class DefaultCustomFieldUserApi implements CustomFieldUserApi {
 
-    private static final Function<CustomFieldModelDao, CustomField> CUSTOM_FIELD_MODEL_DAO_CUSTOM_FIELD_FUNCTION = new Function<CustomFieldModelDao, CustomField>() {
-        @Override
-        public CustomField apply(final CustomFieldModelDao input) {
-            return new StringCustomField(input);
-        }
-    };
+    /**
+     * FIXME-1615 : Cannot replaced by java.util.function.Function because
```

Comment:
For these, we could replace them step by step, e.g. do a single PR to address `DefaultPaginationHelper` across all modules.